scripts/ansible/roles/linux/common/tasks/apt-repo.yml

```diff
@@ -2,11 +2,12 @@
 # Licensed under the MIT License.
 
 ---
+
 - name: Install apt-transport-https APT package
   apt:
     name: apt-transport-https
     state: latest
-  retries: 10
+  retries: 100
```

Comment:
Does the 40th, 50th, etc, try eventually work or does this just delay failure?

Reply:
From what I see in the logs, it usually succeeds before the 30th try, so it does eventually work. I have never seen it fail after setting it to 100 retries, which gives the tasks to wait up to 500 seconds for the apt lock to be released.

Example from a build:
```
    azure-arm: TASK [linux/common : Install apt-transport-https APT package] ******************

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (100 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (99 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (98 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (97 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (96 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (95 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (94 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (93 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (92 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (91 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (90 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (89 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (88 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (87 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (86 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (85 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (84 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (83 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (82 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (81 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (80 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (79 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (78 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (77 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (76 retries left).

    azure-arm: FAILED - RETRYING: Install apt-transport-https APT package (75 retries left).

    azure-arm: [WARNING]: Updating cache and auto-installing missing dependency: python-apt

    azure-arm: changed: [default]

    azure-arm:
    ```